src/web/components/Card/components/CardFooter.tsx

```diff
@@ -54,7 +69,9 @@ export const CardFooter = ({ design, age, mediaMeta, commentCount }: Props) => {
 
 	if (age) {
 		return (
-			<footer className={spaceBetween}>
+			<footer
+				className={isFullCardImage ? fullCardImageAge : spaceBetween}
```

Comment:
Not entirely sure, but does the `spaceBetween` name still make sense when it's an alternative to `fullCardImageAge`?